### DISCUSSION TITLE
1.9.0-3b - htmlexport icon missing

### DISCUSSION THREAD

**Discussion by DhammaNotes** [KosmosisDire/obsidian-webpage-export#563] [General]
Greetings in the 1.9.0-3b beta, the html icon export icon is missing, managed to make a shortcut key though to settings.
Peace

**Comment by KosmosisDire** [comment]
This sounds like a bug report. It is not missing for me so you will need to give more information (prefferably by creating a bug report issue) about how I can reliably reproduce this. Thanks.
